catalyst-gateway/tests/schemathesis-docker-compose.yml

```diff
@@ -28,6 +28,9 @@ services:
   cat-gateway:
     image: cat-gateway:latest
     environment:
+      - DB_URL=postgres://catalyst-event-dev:CHANGE_ME@event-db/CatalystEventDev
+      - CAT_ADDRESS=0.0.0.0:3030
```

Comment:
I dont think this will work, you need to use "127.0.0.1:3030"

Reply:
I just tested this thing, it works fine `0.0.0.0:3030`